fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -579,23 +667,46 @@ struct EncryptedMutationsAndVersionRef {
 	};
 };
 
+class DefineAccumulativeChecksumIndex {
```

Comment:
nit: It is an overkill to use static member functions for these, it is usually used when we need to access private members of the class. I feel the two `const` of `invalidAcsIndex` and `resolverAcsIndex` and one free function of `getCommitProxyAcsIndex()` should be sufficient.